Lesson 12/HW12.cs

```diff
@@ -0,0 +1,46 @@
+namespace HW456.Lesson_12;
+
+/// <summary>
+/// Создать коллекцию из 100 случайных дробей.
+/// Выбрать все дроби, которые являются целым числом.
+/// Вывести на консоль их как целые числа.
+/// </summary>
+
+public class HW12
+{
+    public static void Task()
+    {
+        var list = GenerateListFractions();
+       
+        IEnumerable<Fraction> listInteger = list.Where(x => x.Numerals % x.Denominator == 0); 
+        
+        foreach (var n in listInteger)
+            Console.WriteLine(n.Numerals / n.Denominator);
+    }
+    
+    public static List<Fraction> GenerateListFractions()
+    {
+        Random rnd = new Random();
+        var n = rnd.Next(20);
+        var list = new List<Fraction>();
+
+        for (int i = 0; i < n; i++)
+        {
+            list.Add(new Fraction() {Denominator = rnd.Next(100), Numerals = rnd.Next(100)}); 

```

Comment:
Тут сейчас знаменатель может быть равен нулю, нехорошо это.

```suggestion
            list.Add(new Fraction() {Denominator = 1 + rnd.Next(100), Numerals = rnd.Next(100)}); 

```